prboom2/src/m_menu.c

```diff
@@ -3300,25 +3300,25 @@ static const char *gen_skillstrings[] = {
 
 static const char *gen_compstrings[] =
 {
-  "Default",
-  "Doom v1.2",
-  "Doom v1.666",
-  "Doom/2 v1.9",
-  "Ultimate Doom",
-  "Final Doom",
-  "DosDoom",
-  "TASDoom",
-  "Boom's vanilla",
-  "Boom v2.01",
-  "Boom",
-  "LxDoom",
-  "MBF",
-  "PrBoom 2.03b",
-  "PrBoom 2.1.x",
-  "PrBoom 2.2.x",
-  "PrBoom 2.3.x",
-  "PrBoom 2.4.0",
-  "Latest PrBoom+",
+  "17/Default",
+  " 0/Doom v1.2",
+  "  1/Doom v1.666",
```

Comment:
Uneven spacing?

Reply:
The 1 character is pretty thin, so double spaces are used to fill in the gaps.